app/views/users/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% @user = User.find(params[:id]) %>

```

Comment:
Remove this implementation, it should be done in the controller, then reference the instance variable in the view.